notebooks/samples/explanations/ai-explanations-image-tf2.ipynb

```diff
@@ -0,0 +1,1758 @@
+{
```

Comment:
<h2>Get predictions and explanations</h2><p> </p><p>This section shows how to prepare test images to send to your deployed model, and how to send a batch prediction request to AI Explanations.</p><p> </p><h3>Get and prepare test images</h3><p> </p><p>To prepare the test images:</p><p> </p><ul><li>Download a small sample of images from the flowers dataset -- just enough for a batch prediction.</li><li>Resize the images to match the input shape (192, 192) of the model.</li><li>Save the resized images back to your bucket.</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/192/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>